Danki2/Assets/Scripts/Actor/Services/MovementManager.cs

```diff
@@ -36,6 +36,11 @@ public void ExecuteMovement()
             : Vector3.Normalize(_moveDirection) * speedStat;
 
         _rigidbody.MovePosition(_rigidbody.position + movementVector * Time.deltaTime);
+        if (!movementVector.Equals(Vector3.zero))
+        {
```

Comment:
Nice that it can be done so easily. A bit of smoothing for player rotation would be good as a future task - I'll bring this up next meeting.

Reply:
Yeah for sure, I've added a placeholder card to sprint 5